src/index.rst

```diff
@@ -560,6 +587,10 @@ Change *LSST_VERSION* in the example yaml to what you choose: ::
      inCollection: "HSC/RC2/defaults"
      dataQuery: "exposure = 34342 AND detector = 10"
 
+For different ``butlerConfig`` directory, you also need to grant group permission for PanDA to access the butler::
+
+   $> chmod g+rws /sdf/group/rubin/repo/main/u/<your_operator_name>
```

Comment:
In case someone has already run a `pipetask` command and created an output collection in the repo, probably want that to be a recursive chmod command.

Reply:
fixed with "chmod -R "